src/e2e.js

```diff
@@ -196,6 +201,45 @@ mye2e.prototype.dearmor = function(data) {
   return Promise.resolve(array2buf(e2e.openpgp.asciiArmor.parse(data).data));
 };
 
+// Basic EC Diffie-Hellman shared secret calculation.
+//
+// 'curveName' is a simple string identifying the ECC curve.  "P_256" is
```

Comment:
No, P-256 and 25519 are different curves.

On Wed, Mar 30, 2016 at 1:03 PM, Will <notifications@github.com> wrote:

> In src/e2e.js
> <https://github.com/freedomjs/freedom-pgp-e2e/pull/51#discussion_r57925098>
> :
>
> > @@ -196,6 +201,45 @@ mye2e.prototype.dearmor = function(data) {
> >    return Promise.resolve(array2buf(e2e.openpgp.asciiArmor.parse(data).data));
> >  };
> >
> > +// Basic EC Diffie-Hellman shared secret calculation.
> > +//
> > +// 'curveName' is a simple string identifying the ECC curve.  "P_256" is
>
> it looks like e2e supports Curve25519. is that P_256?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom-pgp-e2e/pull/51/files/1aaa1ccdaa56d8260326866df71a4cd0c5cc9931#r57925098>
>